source.txt

```diff
@@ -281,8 +281,7 @@ Table of Contents
        * 414 if the request URI is too long,
        * 416 if Range requests are supported by the server and the Range
              request can not be satisfied,
-       * 401 for all requests that don't have a bearer token with
-             sufficient permissions,
+       * 401 for all requests that do not have any bearer token,
```

Comment:
Not just that, it's also for revoked, wrong (i.e. unknown to the server), and expired bearer tokens.